externals/global_params.php

```diff
@@ -82,6 +82,6 @@
      $URL_WIKI="https://wiki.gnucash.org/wiki/";
 
      // some defaults for metatags in case the page does not set them
-     $default_title= T_("Free Accounting Software");
-     $default_description= T_("<span class=\"gnucash\">GnuCash</span> is personal and small-business financial-accounting software, freely licensed under the <a href=\"https://www.gnu.org/\">GNU</a> GPL and available for GNU/Linux, BSD, Solaris, Mac OS X and Microsoft Windows.");
+     $default_title= T_("Copylefted Libre Software");
```

Comment:
I think this grade of relabeling should be done after discussion on the gnucash-devel mailing list. IANA

Reply:
I'm not wild about "Copylefted Libre" to replace "Free". Just "Libre" would be better, "Free/Libre" is widely used and might be best of all. Regardless it should say what kind of Free/Libre software it is so "Accounting" should stay.

Reply:
Whatever the order, I think they are all points to make (once).
It similarly already says "Freely licensed" below. I haven't seen Free/Libre anywhere I can think of. Not sure why it would be better.
It is possible to be copyleft without being libre (I think), and it is also something in addition to it, so that makes sense to point out in my head. Explanations along the line of "Use, see, change and share at will; with all." make sense to me.

Reply:
> I haven't seen Free/Libre anywhere I can think of.

[You must not have looked very hard](https://www.gnu.org/philosophy/floss-and-foss.en.html)

> It is possible to be copyleft without being libre

[Nope](https://en.wikipedia.org/wiki/Copyleft).

> it is also something in addition to it

It is indeed, but that's of interest only to people who create derivative works and for them it's sufficient to say that GnuCash is licensed under the GPL.

> Whatever the order, I think they are all points to make (once).

Do you not understand that "Title" and "Description" are complimentary and the latter often repeats with more detail the former?

Reply:
Well I tried looking for it now, and couldn't find anything that isn't some variant of "FLOSS", or FLO… https://wiki.snowdrift.coop/about/free-libre-open

Maybe the point is esoteric, but there could for example be non-commercial copylefted works(?)

I would argue the requirement to share distributed derivative works is of interest to everyone it could benefit. It isn't saying everything is good or even beneficial to all if good in any capacity. The requirement in my view is in the interest of everyone at the level it benefits nobody in particular.

I do understand the relation of title/desc, but would rather keep all redundancy to a minimum and then clarify if need be. No clear inroads on getting to a salient point is being made here. The reader has to already be aware of the duality of "free", which is sure to get lost in translation as a further element to decipher. It adds a flowchart of possibilities rather than narrowing anything down, and that isn't explaining anything well. "Free" is right next to a bank card and some coins, so one would be excused for thinking it is only meant as a monetary point to make.

To Illustrate my near complete ineptitude, I don't even understand that there is some "accounting software" that _isn't_ "financial" (in any way that would leave room for actual confusion). So that amounts to repetition without the panache of linguistic acuity to me.

If it is explained well, I think "copylefted libre software" is the bigger point to make. Right now it isn't even drowning out the idea of money being saved. "Welcome to GnuCash.org" prominent, when it doesn't really do much, or anything over just "Welcome".